kie-archetypes/kie-service-spring-boot-archetype/src/main/resources/archetype-resources/src/main/resources/templates/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE HTML>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title th:text="'Welcome to the ' + ${appName} + ' spring boot app!'"></title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css">
+    <script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js"></script>
+</head>
+<body>
+<nav class="navbar navbar-light bg-light justify-content-between">
+    <a class="navbar-brand" href="#">
+        <span class="fa fa-cogs">&nbsp;&nbsp;</span>
+        <b th:text="${appName} + ' - v' + ${appVersion}"></b>
+    </a>
+</nav>
+<div class="jumbotron text-white font-weight-bold text-center" style="background: url(/images/bg.jpg) no-repeat center center fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;">
+    <h1 class="display-4">Welcome to your new KIE Services App!</h1>
+
+    <p class="lead">App Server has successfully started with following capabilities:
+    <div id="capabilitiesDiv"></div>
+<script>
+    $(document).ready(function() {
+        $.ajax({
+            type: "GET" ,
+            url: "/rest/server" ,
+            dataType: "xml" ,
+            success: function(xml) {
+                var capabilitiesList = xml.getElementsByTagName("capabilities");
```

Comment:
maybe have the capabilities just based on the selected type of appType only - so we would nicely should it has capabilities of business automation or decision management or business optimisation